docs/src/frontend/index.md

```diff
@@ -996,6 +996,23 @@ stdlib.assert(p)
 
 Throws an exception if not given `{!js} true`.
 
+Example:
+
+```js
+load: /examples/argz/index.mjs
+md5: d98c02ebf76e5ce55b6dec475c82539e
+range: 4 - 7
+```
+
+This code checks the arguments passed to the `{!cmd} ./reach run` command.
+Lines 6 and 7 will not throw an exception if `{!cmd} ./reach run index hello "Mr. Postman"` is run in the terminal, because `{!js} process.argv` will return the following `{!rsh} array`:
```

Comment:
```suggestion
Lines 6 and 7 will not throw an exception if `{!cmd} ./reach run index hello "Mr. Postman"` is run in the terminal, because `{!js} process.argv` will return the following array:
```